projects/igniteui-angular/src/lib/slider/slider.component.ts

```diff
@@ -1050,7 +1050,7 @@ export class IgxSliderComponent implements
 
         this._ngZone.runOutsideAngular(() => {
             resizeObservable(this._el.nativeElement).pipe(
-                throttleTime(40),
+                throttleTime(40, undefined, { leading: false, trailing: true }),
```

Comment:
I don't think the slider is related. If there's some issue with its resize observer - log it separately.